contracts/incentives/StakingRewards.sol

```diff
@@ -73,11 +73,18 @@ contract StakingRewards is IRewardDistributionRecipient, ReentrancyGuard {
         duration = (_duration * 24 hours);
 
         rewardDistribution = _avatar;
+
+        // initial notifyRewardsAmount
+        totalRewards = IERC20(_rewardToken).balanceOf(address(this));
+        rewardRate = totalRewards.div(duration);
+        lastUpdateTime = block.timestamp;
+        periodFinish = block.timestamp.add(duration);
+        emit RewardAdded(totalRewards);
     }
 
     uint256 public duration;
 
-    uint256 public initreward;
```

Comment:
Initreward is removed here and nowhere else, it wasn't used anywhere?

Reply:
I replaced it with `totalRewards`